gazebo_ros/test/ros_network/ros_api_checker

```diff
@@ -21,8 +21,9 @@ class Tester(unittest.TestCase):
         cmd = ['rostopic', 'list']
         po = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         out, err = po.communicate()
-        self.assertEqual(po.returncode, 0, 'rostopic failed (%s). stdout: %s stderr: %s'%(cmd, out, err))
-        topics_actual = set(out.split('\n')) - set([''])
+        self.assertEqual(po.returncode, 0, 'rostopic failed (%s). stdout: %s stderr: %s'%(cmd, out.decode(), err.decode()))
```

Comment:
nit, I recommend `subproces.Popen(..., text=True)` instead of adding calls to `decode()` so there are fewer lines changed by this PR

Reply:
Uses `text=True` in 9631b23

Reply:
oops, cleaned up stuff I missed in be09f22